### DISCUSSION TITLE
woocommerce_rest_cannot_view

### DISCUSSION THREAD

**Discussion by kayleejacob1** [XiaoFaye/WooCommerce.NET#768] [General]
{"code":"woocommerce_rest_cannot_view","message":"Sorry, you cannot list resources.","data":{"status":401}}

im sure this issue on my side but tried in postman everything works.  

my code: below

` RestAPI rest = new RestAPI("https://test.com/wp-json/wc/v3/", "testkey", "tespass");
                rest.Debug = true;

                WCObject wc = new WCObject(rest);
                var products = await wc.Product.GetAll();`

**Comment by XiaoFaye** [comment]
Tried to change the value of authorizeheader.

**Reply by kayleejacob1** [reply]
I have tried not setting false, true, false does nothing when i looked at constructer it looks like it is set to true always is there another way to set it?

` RestAPI rest = new RestAPI("https://test.com/wp-json/wc/v3/", "testkey", "tespass",false); - same result
` RestAPI rest = new RestAPI("https://test.com/wp-json/wc/v3/", "testkey", "tespass",true); - same result
` RestAPI rest = new RestAPI("https://test.com/wp-json/wc/v3/", "testkey", "tespass"); - same result

**Comment by kayleejacob1** [comment]
i found error to be in my key
